tools/greenbone-nvt-sync.in

```diff
@@ -522,6 +544,7 @@ do_help () {
   echo " --help         display this help"
   echo " --identify     display information"
   echo " --nvtdir dir   set dir as NVT directory"
+  echo " --notusdir dir   set dir as NOTUS directory"
```

Comment:
```suggestion
  echo " --notusdir dir set dir as NOTUS directory"
```
To have the text aligned with the other entries below and above.

Reply:
Thanks, adapted.